app/assets/stylesheets/content/_modal.sass

```diff
@@ -91,6 +91,12 @@ $ng-modal-image-width:   $ng-modal-image-height
 
   #modal-sorting
     min-height: 120px
+    .form--row
+      overflow: visible
```

Comment:
bollox as in 'shit!' or as in 'boolshit!'? Pardon my lack of briton swearing knowledge